moveit_kinematics/ikfast_kinematics_plugin/scripts/create_ikfast_moveit_plugin.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
```

Comment:
change the filename to be different from the ``create_ikfast_moveit_plugin.py`` so that users understand why there's two. this one seems to include docker and collada conversion, so it should be called something like ``auto_create_ikfast_moveit_plugin.sh``

Reply:
Done.